code/__DEFINES/lazy_templates.dm

```diff
@@ -0,0 +1,13 @@
+#define LAZY_TEMPLATE_KEY_NUKIEBASE "LT_NUKIEBASE"
+#define LAZY_TEMPLATE_KEY_WIZARDDEN "LT_WIZARDDEN"
+#define LAZY_TEMPLATE_KEY_NINJA_HOLDING_FACILITY "LT_NINJAHOLDING"
+#define LAZY_TEMPLATE_KEY_ABDUCTOR_SHIPS "LT_ABDUCTORSHIPS"
+
+#define LAZY_TEMPLATE_KEY_LIST_ALL list( \
+	"Nukie Base" = LAZY_TEMPLATE_KEY_NUKIEBASE, \
+	"Wizard Den" = LAZY_TEMPLATE_KEY_WIZARDDEN, \
+	"Ninja Holding" = LAZY_TEMPLATE_KEY_NINJA_HOLDING_FACILITY, \
+	"Abductor Ships" = LAZY_TEMPLATE_KEY_ABDUCTOR_SHIPS, \
+)
+
+GLOBAL_LIST_INIT(lazy_templates, generate_lazy_template_map())
```

Comment:
`__DEFINES` should only be defines, I know a lot of stuff does this right now but it annoys the hell out of me and regularly gets in the way. This should go either inside a `_globals` file, or with the rest of lazy template code.